queries/elm/highlights.scm

```diff
@@ -0,0 +1,88 @@
+; Keywords
+[
+    "if"
+    "then"
+    "else"
+    "let"
+    "in"
+    (case)
+    (of)
+] @conditional
+
+[
+  (as)
+  (port)
+  (exposing)
+  (alias)
+  (infix)
+  (module)
+] @keyword
+
+[
+  (double_dot)
+  "|"
+] @punctuation.special
+[
+  ","
+  (dot)
+] @punctuation.delimiter
+
+[
+  "("
+  ")"
+  "{"
+  "}"
+] @punctuation.bracket
+
+(type_annotation(lower_case_identifier) @function)
+(port_annotation(lower_case_identifier) @function)
+(function_declaration_left(lower_case_identifier) @function)
+(function_call_expr target:
+  (value_expr) @function)
+
+(value_qid (upper_case_identifier) @constructor)
+(value_qid ((dot) (lower_case_identifier) @field))
+(field_access_expr ((dot) (lower_case_identifier) @field))
+
+(lower_pattern) @parameter
+(record_base_identifier) @method
+
+[
+  (backslash)
+  (underscore)
+] @function
+
+[
+  (operator_identifier)
+  (eq)
+  (colon)
+  (arrow)
+] @operator
+
+(import) @include
+
+(number_constant_expr) @number
+
+(type) @keyword
+
+(module_declaration(upper_case_qid(upper_case_identifier)) @constructor)
+(type_declaration(upper_case_identifier) @constructor)
+(type_ref) @type
+(type_alias_declaration name: (upper_case_identifier) @type)
+(field_type name:
+    (lower_case_identifier) @property)
+
+(union_variant(upper_case_identifier) @symbol)
```

Comment:
would it make sense to have `union_variant` and enum variants in Rust to have the same highlighting? Just a random talk. Sure that `@symbol` also makes sense here.

Reply:
When i started learning, i though of symbols at first. But then indeed the rust enum variants came to mind because these can hold data so i'm not too sure.

Reply:
`@symbol` should be fine.